backend/characterconnect/api/scrabble.py

```diff
@@ -1,5 +1,8 @@
+from ctypes.wintypes import CHAR
 from typing import Dict, Optional, List
 
+from backend.characterconnect.api.game_manager import GameManager
```

Comment:
Having thought about it, I haven't run this yet but I'm almost certain this won't run. I think this is a circular import since the GameManager file imports this, and this file imports game_manager. Python will complain about this